### DISCUSSION TITLE
cards with smaller padding

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3431] [General] [answered]
hey, we have quite a lot of areas in Segment where Card component — but with smaller padding (similar to Visual Picker) — would be perfect to have. This big padding in current Card component is constraining us, especially when we want to use it for smaller things (take a look at the screenshots below).

Did you consider this idea in the past, if now could you consider it now? thanks!

<img width="699" alt="Zrzut ekranu 2023-08-22 o 13 55 13" src="https://github.com/twilio-labs/paste/assets/140410476/98bb4fdf-c1d0-4d6d-b51e-e98e2fc90489">
<img width="653" alt="Zrzut ekranu 2023-08-22 o 13 55 24" src="https://github.com/twilio-labs/paste/assets/140410476/26ac59f5-c3e2-42c6-826c-1d3976822e33">
<img width="398" alt="Zrzut ekranu 2023-08-22 o 13 55 32" src="https://github.com/twilio-labs/paste/assets/140410476/4aa5a551-572c-4da3-bed6-612351c0563a">

**Comment by TheSisb** [comment, marked as answer]
Hi @marektonik,

The Card component accepts a [padding prop](https://paste.twilio.design/components/card#props). This should allow you to adjust the padding for your needs.

**Reply by serifluous** [reply]
^ yep, and now that we've added Figma variables to the components, you'll be able to select a padding size that aligns exactly with what can be built in code.

![Screenshot 2023-08-22 at 10 46 18 AM](https://github.com/twilio-labs/paste/assets/3821840/d5cdbe33-b204-481f-988d-fd5dce94a557)

**Reply by marektonik** [reply]
@TheSisb @serifluous great, thank you!

**Reply by marektonik** [reply]
@serifluous is it also ok to change color variable for Card border/background?

**Reply by serifluous** [reply]
Unfortunately not. If you need a different border/background, you can go the more custom route with the Box primitive: https://paste.twilio.design/primitives/box. In Figma, this would translate to using a rectangle or frame, and applying any border color and background color tokens to it.

What's the use case, though? In most cases, Card _should_ work just fine.

**Reply by marektonik** [reply]
@serifluous we were thinking about 'disabled' or 'error' style for a card. 

![Zrzut ekranu 2023-08-28 o 09 25 49](https://github.com/twilio-labs/paste/assets/140410476/65fabc8d-0743-4fc4-bde4-2b6be87d04f0)

**Reply by SiTaggart** [reply]
Hi there

You can't disable text based content, only interactive controls. Text must also remain with a 4.5:1 color contrast to it's background.

Thanks
